package.json

```
@@ -20,6 +20,7 @@
     "android",
     "ios",
     "js",
+     "windows",
```

```suggestion
    "windows",
```
(indentation didn't line up)